### ISSUE TITLE
[$500] [HOLD for payment 2024-03-26] [Simplified Collect][Taxes] Create WorkspaceTaxesSettingsPage

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#37788]
Implement [this section](https://docs.google.com/document/d/12FZycslrIE95QoKM8DZ37H_wLbH9w1jt3uA-a6BCqtc/edit#bookmark=id.2kcxx3umdfhl) of the Simplified Collect: Taxes doc

cc @kosmydel @filip-solecki

<details><summary>Issue Owner</summary>Current Issue Owner: @anmurali</details>

**Comment by filip-solecki** [comment]
Hi! I am Filip from SWM and I'd like to work on this issue!

**Comment by luacmartins** [comment]
on hold

**Comment by DylanDylann** [comment]
No longer on hold. I would be delighted if I could continue to accompany @filip-solecki on this issue

**Comment by DylanDylann** [comment]
@luacmartins Also help to remove the hold label from the title of the issue

@filip-solecki It seems these features are our priority. Let's start making the PR for this one

**Comment by filip-solecki** [comment]
I am preparing PR right now

**Comment by DylanDylann** [comment]
@luacmartins @mountiny We handled 4 issues in [this PR](https://github.com/Expensify/App/pull/38373) that make PR become bigger. I want to ask about how we process payments here

cc @anmurali

**Comment by luacmartins** [comment]
Thanks for reaching out @DylanDylann! We'd still pay for a single PR review, since that's what was done here. While I understand that not all issues and PRs are made equal, and some are bigger or more complex than others, I still think that the compensation is fair in this situation.

**Comment by DylanDylann** [comment]
@luacmartins Thank you for your response. I am happy with your decision. This is [a similar case](https://github.com/Expensify/App/pull/38056) that we also should consolidate into an issue 

cc @puneetlath

**Comment by mountiny** [comment]
I believe this is $500 to @DylanDylann and we can close it

**Comment by luacmartins** [comment]
Bump for payment

**Comment by luacmartins** [comment]
Waiting on payment @anmurali

**Comment by anmurali** [comment]
Offer is [here](https://www.upwork.com/nx/wm/offer/101798289)

**Comment by DylanDylann** [comment]
@anmurali Accepted offer. Thanks

**Comment by luacmartins** [comment]
Seems like we're all done here? @DylanDylann

**Comment by DylanDylann** [comment]
@luacmartins I accepted the offer. Waiting for BZ pay the offer

**Comment by luacmartins** [comment]
Thanks for the update. Just waiting on @anmurali

**Comment by anmurali** [comment]
@DylanDylann has been paid.
